test/ess-tests.el

```diff
@@ -0,0 +1,86 @@
+;; ess-tests.el --- Tests for ESS
+;;
+;;
+;; Filename: ess-tests.el
+;; Created: 07-05-2015 (ESS 15.09)
+;; Keywords: tests, indentation
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; This file is *NOT* part of GNU Emacs.
+;; This file is part of ESS
+;;
+;; This program is free software; you can redistribute it and/or
+;; modify it under the terms of the GNU General Public License as
+;; published by the Free Software Foundation; either version 3, or
+;; (at your option) any later version.
+;;
+;; This file is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; A copy of the GNU General Public License is available at
+;; http://www.r-project.org/Licenses/
+;;
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;;; Commentary:
+;;
+;; To run these tests:
+;;   All tests: M-x ert t
+;;
+;; To apply styles for manual testing:
+;;   M-: (let ((ess-style-alist ess-test-style-alist))
+;;         (ess-set-style 'test-misc1))
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;;; Code:
+
+(require 'ert)
+
+
+;;; Indentation tests
+
+(defvar ess-test-style-alist
+  ;; New test-RRR style containing additional settings.
+  ;; Compared to RRR, it makes sure personal configuration does not
+  ;; interfere with tests
+  `((test-RRR
+     ,@(cdr (assq 'RRR ess-style-alist))
+     (ess-brace-imaginary-offset . 0)
+     (ess-continued-brace-offset . '(0))
+     (ess-close-paren-offset . 0)
+     (ess-fancy-comments . t))
+    (test-misc1
```

Comment:
It's not completely random, I tried to make it as different from RRR as possible.
Hopefully we'll come up with a better strategy later.

I was thinking the same about the Rstudio style.